components/cam/src/physics/crm/samxx/test/build/download_data.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+wget https://code.ornl.gov/imn/data/raw/master/crmdata_nx8_ny8_nz28_nxrad2_nyrad2.nc
+wget https://code.ornl.gov/imn/data/raw/master/crmdata_nx32_ny1_nz28_nxrad2_nyrad1.nc
```

Comment:
What's the point of this file?

Reply:
To grab data needed for standalone tests of samxx